src/files/BrsFile.spec.ts

```diff
@@ -1311,6 +1311,25 @@ describe('BrsFile', () => {
             expect(file.functionCalls[1].nameRange).to.eql(Range.create(5, 20, 5, 23));
         });
 
+        it('finds function calls that are unfinished', () => {
+            let file = new BrsFile('absolute_path/file.brs', 'relative_path/file.brs', program);
+            file.parse(`
+                function DoA()
+                    DoB("a"
+                end function
+                function DoB(a as string)
+                    DoC(
+                end function
+            `);
+            expect(file.functionCalls.length).to.equal(2);
```

Comment:
Can you add a `expectDiagnostics()` call here with all the diagnostics you think this should produce?

Reply:
done